src/ts/app/root/src/render.ts

```diff
@@ -0,0 +1,38 @@
+import {html, TemplateResult} from "lit-html";
+
+export function render(): TemplateResult {
+    return html`
+        <style>
+            .main {
```

Comment:
can we move this to css files? I guess I might have missed this part of the initial framework review. CSS is scoped, but it it always inline in the HTML? Debugging CSS sucks, but it's really nice when you have things in seperate files because you see the higharchy and file names in the dev tools.